src/System.ValueTuple/src/System/ValueTuple/ValueTuple.cs

```diff
@@ -585,7 +601,17 @@ int ITupleInternal.GetHashCode(IEqualityComparer comparer)
         /// </remarks>
         public override string ToString()
         {
-            return "(" + Item1 + ", " + Item2 + ")";
+            return "(" + ToStringEnd();
+        }
+
+        string ITupleInternal.ToStringEnd()
+        {
+            return ToStringEnd();
+        }
+
+        private string ToStringEnd()
+        {
+            return Item1 + ", " + Item2 + ")";
```

Comment:
I agree it's unfortunate. I'll duplicate the code and add the element-wise `ToString` as you suggested.